frontend/talentsearch/src/js/components/loggedOut/LoggedOutPage.tsx

```diff
@@ -108,7 +108,7 @@ const LoggedOutPage: React.FC = () => {
         centered
         isOpen={loggedIn}
         onDismiss={() => {
-          navigate(talentPaths.profile());
+          navigate(talentPaths.home());
```

Comment:
Navigating to the profile page here would now require we look up the signed in users id and have a fallback URL in case the look-up fails. Where do we want this to navigate to?

Reply:
@Jerryescandon When a user navigates directly to the logged out page while currently logged in we show them this dialog:
![image](https://user-images.githubusercontent.com/49166751/183510668-b4d54ed5-6c87-44c4-98ac-c47fc9d7530e.png)
If they click the cancel button they are navigated to the tc-report homepage but if they hit escape or the X in the corner we instead navigate them to their profile. Where do we actually want to send the user if they opt not to sign out?

Reply:
If think the homepage will be as good place as any if we don't know who they are, and they came straight to this page. 

My Profile would probably be the best case if they are signed in already tho.

Reply:
Ok, we only show this if they are signed in so I'll have it navigate to their profile then. Thanks! ❤️